### DISCUSSION TITLE
Run flow tests using Salesforce CLI

### DISCUSSION THREAD

**Discussion by nvuillam** [forcedotcom/cli#2492] [Ideas]
It's great that we can create Test Flows , but how can we run them during CI/CD pipeline checks ?

I did not find any sf command, is there one ?

If not, is it planned ?

**Comment by mshanemc** [comment]
There's currently no API to call for that.  When that gets built, we could definitely create a command to automate it.

What would you call it?

**Reply by nvuillam** [reply]
I think something similar than sfdx force:apex:test:run , so sfdx force:flow:test:run , with similar arguments ? :) ( and of course following sf cli style, but i don't masterize it yet ^^ )

edit:  `sf flow run test` ? :)

https://developer.salesforce.com/docs/atlas.en-us.sfdx_cli_reference.meta/sfdx_cli_reference/cli_reference_apex_commands_unified.htm#cli_reference_apex_run_test_unified

**Comment by mitchspano** [comment]
This has been delivered with the [`sf flow run test` command](https://developer.salesforce.com/docs/atlas.en-us.sfdx_cli_reference.meta/sfdx_cli_reference/cli_reference_flow_commands_unified.htm#cli_reference_flow_run_test_unified) and the [`sf logic run test` command](https://developer.salesforce.com/docs/atlas.en-us.sfdx_cli_reference.meta/sfdx_cli_reference/cli_reference_logic_commands_unified.htm#cli_reference_logic_run_test_unified
).
